boilerplates/react-union-boilerplate-liferay-basic/hero-portlet/src/main/java/eu/reactunion/boilerplate/configuration/HeroConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package eu.reactunion.boilerplate.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+import eu.reactunion.boilerplate.constants.HeroPortletKeys;
+
+/**
+ * Configuration class of the Hero portlet.
+ *
+ * @author Roman Srom (roman.srom@lundegaard.eu)
+ */
+@ExtendedObjectClassDefinition(
+        category = "ReactUnion",
```

Comment:
I would prefer to use one of the predifined categories than creating new for react uinon (e.g. Foundation).

Reply:
Implemented